scripts/monte_carlo.py

```diff
@@ -158,6 +172,33 @@ def monte_carlo_sampling_scipy(
     return lh
 
 
+def single_best_in_worst_list(worst_list, best_list):
+    """
+    Return list with single best value per list of worst values
```

Comment:
Please, may you increase the description?
This is needed in the api_reference so it makes sense to improve.